RadarSDK/RadarEvent.m

```diff
@@ -38,6 +38,22 @@ @implementation RadarEvent
     return mutableEvents;
 }
 
++ (RadarEvent *_Nullable)eventFromObject:(id _Nonnull)object {
+    if (!object || ![object isKindOfClass:[NSDictionary class]]) {
+        return nil;
+    }
+
+    NSDictionary *eventObj = (NSDictionary *)object;
+
+    RadarEvent *event = [[RadarEvent alloc] initWithObject:eventObj];
+
+    if (!event) {
+        return nil;
+    }
+
+    return event;
```

Comment:
`return event` is sufficient. Better yet, just `return [[RadarEvent alloc] initWithObject:eventObj];`.

Reply:
good looks, much cleaner